src/commands/prompt/commands.ts

```diff
@@ -58,6 +64,43 @@ export function processCommand(input: string, context: CommandContext): boolean
     return true;
   }
 
+  if (command === '/save') {
+    if (context.messages.length === 0) {
+      output.outputInfo('There are no messages to save.');
+      return true;
+    }
+
+    try {
+      saveConversation(context, args);
+      return true;
+    } catch (error) {
+      output.outputError(error);
+      return true;
+    }
+  }
+
   output.outputError(`Unknown command: ${command} ${args.join(' ')}`);
   return true;
 }
+
+function saveConversation(context: CommandContext, args: string[]) {
+  const conversation = context.messages.reduce((prev, current) => {
```

Comment:
[nit] Pls avoid using `reduce` as it makes code harder to read. just use regular loop or `foreEach`

Reply:
done